pages/dkp/konvoy/2.1/choose-infrastructure/azure/advanced/new/index.md

```diff
@@ -89,9 +118,13 @@ enterprise: false
     > ${CLUSTER_NAME}.yaml
     ```
 
+The output of this command is `${CLUSTER_NAME}.yaml` file that you can examine or modify further. If you are using this method to create your cluster without setting existing network infrastructure values, review the information under "Inspect or edit the cluster objects" and then go to "Create the actual Kubernetes cluster."
```

Comment:
I don't think the last sentence adds much info here, I'd suggest instead of adding that extra info here, we just add something to the "configure existing network infrastructure" below that says "if you don't need this, you can skip it".  See below

```suggestion
The output of this command is `${CLUSTER_NAME}.yaml` file that you can examine or modify further. 
```

Reply:
Resolved as noted in my local branch.